src/main/groovy/org/pih/warehouse/core/HistoryItem.groovy

```diff
@@ -0,0 +1,15 @@
+package org.pih.warehouse.core
+
+class HistoryItem<T> {
+    String identifier
+
+    Date date
+
+    // FIXME: I didn't know what name suits the best for the location
+    Location associatedLocation
```

Comment:
Let's just call this location for now. Or facility if we want to be specific about what we're talking about.